new_lineage_panel/src/App.tsx

```diff
@@ -158,19 +126,22 @@ function App() {
     table: "",
     sessionId: "",
   });
+  const [leftExpansion, setLeftExpansion] = useState(0);
+  const [rightExpansion, setRightExpansion] = useState(0);
   const [collectColumns, setCollectColumns] = useState<
     Record<string, string[]>
   >({});
-  const [confidence, setConfidence] = useState<{
-    confidence: string;
-    operator_list?: string[];
-  }>({ confidence: "high" });
+  const [confidence, setConfidence] = useState<Confidence>({
+    confidence: "high",
```

Comment:
if the value for this confidence will be set of strings like high or low only, using it as an enum could be better